tox.ini

```diff
@@ -46,12 +46,13 @@ extras =
 commands_pre =
     find {toxinidir} -type f -not -path '{toxinidir}/.tox/*' -path '*/__pycache__/*' -name '*.py[c|o]' -delete
 commands =
-    unit: pytest test/unit/ --cov={toxinidir}/molecule/ --no-cov-on-fail {posargs}
-    functional: pytest test/functional/ {posargs}
-    lint: flake8
-    lint: yamllint -s test/ molecule/
+    unit: python -m pytest test/unit/ --cov={toxinidir}/molecule/ --no-cov-on-fail {posargs}
+    functional: python -m pytest test/functional/ {posargs}
+    lint: python -m flake8
+    lint: python -m yamllint -s test/ molecule/
 whitelist_externals =
     find
+    rm
```

Comment:
If you add it here, remove `whitelist_externals` in the last section.

Reply:
By the way, what causes a call to `rm` here?

Reply:
`build-dists` does use `rm`.

Reply:
But true, if I fix it I fix it completly. Already pushed new version.
BTW, pur already released a fix after I raised the bug https://github.com/alanhamlett/pip-update-requirements/issues/24 --- we no longer need it but it was really nice have it fixed and released so quickly.